app/routes/home.js

```diff
@@ -44,6 +54,9 @@ export default Ember.Route.extend({
   actions: {
     onLoadBooks(sortOrder, skip, max) {
 
+      // I dont understand this flow. I would expect data loading to be handled
```

Comment:
refreshModel has a bad user experience where the loading comes in the entire template rather than only the table section of the template @artzte